product_docs/docs/migration_portal/3.5.0/04_mp_migrating_database/01_mp_schema_extraction.mdx

```diff
@@ -0,0 +1,210 @@
+   
+   ```
+   $ cat parameters2.par
+
+   EXCLUDE=SCHEMA:"IN(select username as schema_name from sys.dba_users where LOWER(username) LIKE 'pg_%' OR username IN
+   ('ANONYMOUS','APEX_PUBLIC_USER','APEX_030200','APEX_040000','APEX_040200','APPQOSSYS','AUDSYS','CTXSYS','DMSYS','DBSNMP',
+   'DBSFWUSER','DEMO','DIP','DMSYS', 'DVF','DVSYS','EXFSYS','FLOWS_FILES','FLOWS_020100', 'FRANCK','GGSYS','GSMADMIN_INTERNAL',
+   'GSMCATUSER','GSMROOTUSER','GSMUSER','LBACSYS','MDDATA','MDSYS','MGMT_VIEW','OJVMSYS','OLAPSYS','ORDPLUGINS','ORDSYS',
+   'ORDDATA','OUTLN','ORACLE_OCM','OWBSYS','OWBYSS_AUDIT','PDBADMIN','RMAN','REMOTE_SCHEDULER_AGENT','SI_INFORMTN_SCHEMA',
+   'SPATIAL_CSW_ADMIN_USR','SPATIAL_WFS_ADMIN_USR','SQLTXADMIN','SQLTXPLAIN','SYS$UMF','SYS','SYSBACKUP','SYSDG','SYSKM',
+   'SYSRAC','SYSTEM','SYSMAN','TSMSYS','WKPROXY','WKSYS','WK_TEST','WMSYS','XDB','XS$NULL'))"
+   ```
+   The attributes and values in the above command specify the following options:
+   + `EXCLUDE` specifies which schemas to exclude. Append a list of the [Unsupported schemas](#unsupported-schemas) to ensure that the Data Pump utility *does not* extract any system schemas (including schemas beginning with **pg_**).
+
+6. To generate a SQL file from the dump file, run the import command. 
+
+   For example, to generate **YourSchemas.SQL** file from the db.dump file, enter the following command:
+ 
+   ```
+    $ impdp <Username>@<ConnectIdentifier> DIRECTORY=<directory_name> TRANSFORM=OID:n SQLFILE=YourSchemas.sql DUMPFILE=db.dump
+   ```
+
+
+## Supported schemas and objects
+
+See the following sections for supported and unsupported schemas and objects.
+
+### Unsupported schemas
+
+See the following table for a list of Oracle systems schemas you must exclude while generating the SQL dump file.
+
+|                       |                     |                   |                    |                         |
+| :-------------------- | :------------------ | :---------------- | :----------------- | :---------------------- |
+| ANONYMOUS	| APEX_PUBLIC_USER	| APEX_030200	| APEX_040000	| APEX_040200	 |
+| APPQOSSYS	| AUDSYS	| CTXSYS	| DMSYS	| DBSNMP	 |
+| DBSFWUSER	| DEMO	| DIP	| DMSYS	| DVF	 |
+| DVSYS	| EXFSYS	| FLOWS_FILES	| FLOWS_020100	| FRANCK	 |
+| GGSYS	| GSMADMIN_INTERNAL	| GSMCATUSER	| GSMROOTUSER	| GSMUSER	 |
+| LBACSYS	| MDDATA	| MDSYS	| MGMT_VIEW	| OJVMSYS	 |
+| OLAPSYS	| ORDPLUGINS	| ORDSYS	| ORDDATA	| OUTLN	 |
+| ORACLE_OCM	| OWBSYS	| OWBYSS_AUDIT	| PDBADMIN	| RMAN	 |
+| REMOTE_SCHEDULER_AGENT	| SI_INFORMTN_SCHEMA	| SPATIAL_CSW_ADMIN_USR	| SPATIAL_WFS_ADMIN_USR	| SQLTXADMIN	 |
+| SQLTXPLAIN	| SYS$UMF	| SYS	| SYSBACKUP	| SYSDG	 |
+| SYSKM	| SYSRAC	| SYSTEM	| SYSMAN	| TSMSYS	 |
+| WKPROXY	| WKSYS	| WK_TEST	| WMSYS	| XDB	 |
+| XS$NULL	| 				 |	
+
+!!! Note
+    EDB Postgres Advanced Server does not support schema names starting with **pg_**, which should also be excluded while extracting from Oracle.
+
+### Supported object types
```

Comment:
I'd list the supported types first to align with the section title and intro sentence (and it is generally better to start with the positive case)